src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js

```diff
@@ -0,0 +1,63 @@
+(function(global, doc, eZ) {
+    let sectionsCords = {};
+    const scrollOffset = 100;
+    const isVerticalScrollVisible = () => {
+        return doc.body.scrollHeight > global.innerHeight;
+    };
+    const showSection = (anchorTargetSection) => {
+        doc.querySelectorAll('.ibexa-anchor-navigation-menu__btn').forEach((btn) => {
+            btn.classList.toggle('ibexa-anchor-navigation-menu__btn--active', btn.dataset.anchorTargetSection === anchorTargetSection);
+        });
+
+        doc.querySelectorAll('.ibexa-anchor-navigation-sections__section').forEach((section) => {
+            const { anchorSectionId } = section.dataset;
```

Comment:
What would you say for some consistency? Either destruct also in previous loop `btn.dataset`, or not do it here.